invest/templates/invest/high-potential-opportunity-detail.html

```diff
@@ -0,0 +1,231 @@
+{% extends 'core/base.html' %}
+{% load static from staticfiles %}
+
+{% load add_export_elements_classes from directory_components_tags %}
+{% block head_css %}
+    {{ block.super }}
+    <link href="{% static 'styles/high-potential-opportunity-detail.css' %}" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block head_title %}
+    High Potential Opportunities - great.gov.uk
+{% endblock %}
+
+{% block content %}
+<section id="hero" style="background-image: url({{ page.hero_image.url }})">
+    <div class="container">
+        <p class="breadcrumbs body-text">
```

Comment:
components has a breadcrumbs template now https://github.com/uktrade/directory-components/blob/master/directory_components/templates/directory_components/breadcrumbs.html

Reply:
good stuff :)

food for thought: the breadcrumbs component expects `parent_site` dictionary, which would mean passing `parent_site={'url': '/', 'title': 'Home'}` from the view to the template, which is a bit more complex than it needs to be.

looks like we could update the breadcrumb component to something like
`<a href="{{ root_page_url }}">{{ root_page_breadcrumb_label }}</a>`

Reply:
oh oh. the breadcrumb component is using `page.title`. `title` is the cms title :(

I'll use the current html, and tomorrow will contribute to directory components to allow those values to be passed in, rather than pulling them from a dictionary (which is a bit restrictive)

Reply:
yeah this component needed some tweaking so these are good improvements to add